pkg/kubelet/kubelet.go

```diff
@@ -962,7 +963,10 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	if utilfeature.DefaultFeatureGate.Enabled(features.SystemdWatchdog) {
 		// NewHealthChecker returns an error indicating that the watchdog is configured but the configuration is incorrect,
 		// the kubelet will not be started.
-		klet.healthChecker, err = watchdog.NewHealthChecker(klet)
+		checkers := []healthz.HealthChecker{
+			klet.containerManager.GetPluginRegistrationServerChecker(),
```

Comment:
why only for systemd watchdog and not for `/healthz`?

Reply:
I will to add it to `/healthz`.